.claude/agents/nvim-markview-specialist.md

```
@@ -0,0 +1,366 @@
+- Rendering features guide
+- Performance tips
+- Customization examples
+
+## 5. Few-Shot Examples
+### ✅ Good Example: Comprehensive markview.nvim configuration
+```lua
+return {
+  "OXY2DEV/markview.nvim",
+  lazy = false,
+  dependencies = {
+    "nvim-treesitter/nvim-treesitter",
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)